tests/basic_setters.rs

```diff
@@ -0,0 +1,38 @@
+#[macro_use] extern crate custom_derive;
+#[macro_use] extern crate derive_builder;
+
+custom_derive!{
+    #[derive(Debug, PartialEq, Default, Builder)]
+    struct Lorem {
+        ipsum: String,
+        dolor: String,
+        sit: i32,
+        amet: bool,
+    }
+}
+
+impl Lorem {
+    pub fn new<T: Into<String>>(value: T) -> Self {
```

Comment:
Yeah, you can write all the `impl`s you want :)

> But this would collide, if we reserved fn new() for ourselves.

Yep. Which is why I don't want us to write a `fn new`. This way we don't need to deal with required fields and people can always just do their own thing. (And maybe later switch to a new version of this crate that does more stuff for them.)